website/docs/secrets/manage-secrets-ui.mdx

```diff
@@ -31,13 +31,49 @@ Here, you will be prompted to enter the `External Secret Name` and the `Target K
 
 It's important to note that the chosen `SecretStore` may be a cluster-scoped `SecretStore` ie: `ClusterSecretStore` or a namespace-scoped `SecretStore`.
 
+If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
+
 ![Create new Secret](./imgs/create-secret-2.png)
 
-If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
+If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.
 
 ![Create new Secret](./imgs/create-secret-3.png)
 
-If you choose a cluster-scoped `ClusterSecretStore`, the new secret will be created in a namespace of your choice.
+#### Adding Secret Properties
+
+After you have choose your desired `SecretStore` & `SecretPath` the UI allows you to add secret properties in two different scenarios:
+
+##### 1. Adding and Removing Multiple Property Fields
+
+The first scenario allows you to add or remove multiple property fields. Each added `property` also has an optional `SecretKey` field. Here's how to do it:
+
+In the `Properties` section, click the `Add` button to create a new property field.
+
+![Add Property](./imgs/add-property-1.png)
+
+Enter the name of the `property` you want to create. You can add as many properties as you need.
+
+![Enter Property](./imgs/enter-property-1.png)
+
+If you wish to specify a `SecretKey` for the property, enter it in the `SecretKey` field. If this field is left blank, the `property` name will be used as the `SecretKey`.
+
+[Add SecretKey](./imgs/add-secret-key-1.png)
+
```

Comment:
It could be good to extend the piece of how the example translates as `Kubernetes Secret` to provide the complete example or a link for that example (for example to original ESO website)